no-vary-search.bs

```diff
@@ -251,7 +280,7 @@ Two [=URLs=] |urlA| and |urlB| are <dfn export>equivalent modulo search variance
   <table>
     <thead>
       <tr>
-        <th>Equivalent URLs
+        <th>Equivalent URL strings
```

Comment:
URLs themselves in their parsed form cannot have non-ASCII query components. It's only the parsed query list of (key, value) pairs that contain potentially-non-ASCII strings. But, people can send non-ASCII inputs to the URL parser, which is what I think is valuable to exemplify here and in the newly-added example.